purism/librem/5r4/kernel/6.1.10.nix

```
@@ -0,0 +1,14 @@
+{ buildLinux, fetchFromGitLab, ... }@args:
+buildLinux (args // rec {
+  defconfig = "librem5_defconfig";
+  version = "6.1.10-librem5";
+  modDirVersion = version;
+  src = fetchFromGitLab {
+    domain = "source.puri.sm";
+    owner = "Librem5";
+    repo = "linux";
+    rev = "pureos/6.1.10pureos1";
+    hash = "sha256-Cc16vMUcJ/a2k3zMynqZ99t1LyTSs7EXKdNGF6OTS1s=";
+  };
+  kernelPatches = [ ];
```

```suggestion
  kernelPatches = [ ];
  structuredExtraConfig = with lib.kernel; {
    # buildLinux overrides this and defaults to 32, so go back to the value defined librem5_defconfig
    # this is required for millipixels to take photos, otherwise the VIDIOC_REQ_BUFS ioctl returns ENOMEM
    CMA_SIZE_MBYTES = lib.mkForce (freeform "320");
  };
```

See the upstream `librem5_defconfig` at https://source.puri.sm/Librem5/linux/-/blob/pureos/6.4.5pureos1/arch/arm64/configs/librem5_defconfig#L1178.